test/regress/cli/regress0/cores/unsat-core-lemmas.smt2

```diff
@@ -0,0 +1,37 @@
+; COMMAND-LINE: --simplification=none --unsat-cores-mode=sat-proof
+; EXPECT: unsat
+; EXPECT: (
+; EXPECT: a0
```

Comment:
I would omit the names and just expect 
```
(
)
```

Reply:
Or just conjoin all assertions in the unsat core and have a single name expected.

Reply:
I do not understand.

Reply:
This benchmark output is sensitive to the order of names in the unsat core, I would design the test to not rely on this.